src/main/java/org/spongepowered/api/item/recipe/Recipe.java

```diff
@@ -47,12 +47,12 @@
 public interface Recipe {
 
     /**
-     * Returns the list of item types that result when successful crafting of
-     * this Recipe is completed.
+     * Returns the item type result when successful crafting of this Recipe
+     * is completed.
      *
-     * @return The resultant list of item types
+     * @return The resultant item type
      */
-    List<ItemType> getResultTypes();
+    ItemType getResultType();
```

Comment:
Recipes can have more than one output, this was established a long time ago
https://github.com/SpongePowered/SpongeAPI/pull/242#issuecomment-64929509